fastlane/lib/fastlane/actions/docs/sync_code_signing.md

```diff
@@ -322,9 +322,28 @@ You'll be asked for the new password on all your machines on the next run.
 If you want to manually decrypt a file you can.
 
 ```no-highlight
-openssl aes-256-cbc -k "<password>" -in "<fileYouWantToDecryptPath>" -out "<decryptedFilePath>" -a -d
+openssl aes-256-cbc -k <password> -in <fileYouWantToDecryptPath> -out <decryptedFilePath> -a -d
```

Comment:
Can we keep the quotes in here? The quotes don't hurt but quotes can be needed if the file paths have spaces in them